packages/block-library/src/navigation-menu/index.php

```diff
@@ -65,6 +65,11 @@ function build_css_colors( $attributes ) {
  * @return string Returns the post content with the legacy widget added.
  */
 function render_block_navigation_menu( $attributes, $content, $block ) {
+	$classes = 'wp-block-navigation-menu';
+	if ( ! empty( $attributes['className'] ) ) {
+		$classes = $classes . ' ' . $attributes['className'];
```

Comment:
Minor: this might be a style guide thing but why not: 

```php
$classes .= ' ' . $attributes['className'];
```

It's a super minor comment though. Sorry!